### ISSUE TITLE
[docs] enumerate some of the common issues with AWS

### ISSUE THREAD

**Issue by errordeveloper** [weaveworks/eksctl#172]
We need to document a few key areas:

-  CloudFormation
  - mention that it is the go-to place for troubleshooting (we have this in the output, but need to document in plain English also)
  - mention that it owns all resource, and if one needs to modify something, they need to know what they are doing, and mention that they can hack the template and update stack if they really have to do that
  - mention what stacks we have and what they are
  - mention what stack rollback error means
- mention that limits are a thing
  - note default VPC and EKS limits
  - link to page for raising limits
- mention us-east-1 availibility error and what to do about it
- mention ELB isseue that applies to new accounts:
  - @christopherhein: "If you haven't provisioned an ELB before and try to use K8s to create an elb via a service for some reason we don't automatically configure `iam:CreateServiceLinkedRole` the easiest way to fix is create an elb outside of k8s then once it provisions delete it and then try again in k8s"

**Comment by nukepuppy** [comment]
defining the CIDR range of a net new VPC + subnet ranges should be a pretty big one in general.. ?

**Comment by bowlesns** [comment]
Service Limits page: https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html
Elastic IP limit should be added as well

**Comment by Himangini** [comment]
Closing this issue due to lack of activity.
